### DISCUSSION TITLE
How does access control currently  work in superset and what's the plan?

### DISCUSSION THREAD

**Discussion by DataScientistSamChan** [apache/superset#18928] [Ideas]
I scroll through all the security-related issues here(some listed: https://github.com/apache/superset/issues/10408, https://github.com/apache/superset/issues/17914, https://github.com/apache/superset/issues/17913). Currently, I only see there is only rbac support at the dashboard level, but really in a big corporation, rbac support should be available in all levels(e.g. Database, datasets, slices).  I wonder what's the plan on access control management. Afaik,  Airbnb has implemented a robust security manager, as mentioned in this article(https://medium.com/airbnb-engineering/supercharging-apache-superset-b1a2393278bd), I wonder if it is possible for  them to share this solution too?

**Comment by srinify** [comment]
Hey there! We use Github issues in this community to discuss bugs, so I'm moving this over to Discussions :]
